index.html

```diff
@@ -798,6 +798,37 @@ <h3>Bitstring Expansion Algorithm</h3>
 
   </section>
 
+  <section class="informative">
+    <h2>Media Types</h2>
+    <p>
+      When dereferencing <code>statusListCredential</code>,
+      the content type of the <code>statusListCredential</code> might
+      be any media type registered for the purpose of expressing a
+      verifiable credential with one or more proofs.
+    </p>
+    <p>
+      For example, a verifiable credential secured with 
+      Data Integrity Proofs might have content type <code>application/vc+ld+json</code>,
+      whereas a verifiable credential secured with 
+      SD-JWT might have content-type <code>application/vc+ld+json+sd-jwt</code>.
```

Comment:
`application/vc+ld+json+sd-jwt` should not exist. `application/sd-jwt` may contain `application/vc+ld+json`.
```suggestion
      SD-JWT might have content-type <code>application/sd-jwt</code>.
```